apps/examples/testcase/le_tc/kernel/tc_irq.c

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_irq.c
+
+/// @brief Test Case Example for fin_wait API
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <sys/types.h>
+#include <tinyara/timer.h>
+#include <tinyara/irq.h>
+#include "tc_internal.h"
+
+#  define TIMER_DEVNAME  "/dev/timer%d"
+
+/**
+ * @fn                  :tc_irq
+ * @brief               :test the fin_wait using timer
+ * @scenario            :none
+ * API's covered        :fin_wait
+ * Preconditions        :none
+ * Postconditions       :none
+ * @return              :void
+ */
+
+static void tc_irq_fin_wait(void)
+{
+	struct timer_notify_s notify;
+	notify.arg   = NULL;
+	notify.pid   = (pid_t)getpid();
+	int intval = 100;
```

Comment:
Please declare this value through `#define INTERVAL`.
Then we can test with different interval easily.